README.md

```diff
@@ -1,10 +1,18 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/driver-registrar.svg?branch=master)](https://travis-ci.org/kubernetes-csi/driver-registrar)
 # Cluster Driver Registrar
 
-A sidecar container that
+A controller sidecar container that
 
 1. Registers the containerized CSI driver with Kubernetes and create a CSIDriver object
 
+This sidecar container is only needed if you need the following Kubernetes
+features:
```

Comment:
maybe add a link on how to "activate/use" these features if the user reading this is interested